manifests/a/Andersama/obs-asio/3.2.1/Andersama.obs-asio.installer.yaml

```diff
@@ -0,0 +1,25 @@
+# Created with komac v2.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Andersama.obs-asio
+PackageVersion: 3.2.1
+InstallerLocale: en-US
+InstallerType: inno
+Scope: machine
+InstallerSwitches:
+  Silent: /VERYSILENT
+  SilentWithProgress: /SILENT
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: OBSProject.OBSStudio
+ProductCode: '{77AFA351-2AD0-4997-BBA8-1B1285074A8A_is1'
+ReleaseDate: 2024-05-21
+AppsAndFeaturesEntries:
+- ProductCode: '{77AFA351-2AD0-4997-BBA8-1B1285074A8A_is1'
+ElevationRequirement: elevatesSelf
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/Andersama/obs-asio/releases/download/v3.2.1f/obs-asio-3.2.1-windows-x64-Installer.exe

```

Comment:
> Possible duplicate package entry.
> 
> Similar installer SHA256 hash found in manifest:
> 
> * `InstallerSha256: ab75f582c8ecaf1ddd288dfcc7076654779874e4425bf42d34545b316da0d5ce`
>   
>   * manifests\a\Andersama\obs-asio\3.2.1f


Hi @PckgrBot,

Is this the same version? It has the same version number in the file path.